src/main/api/2/studio-api-2.yaml

```diff
@@ -2911,6 +2913,163 @@ paths:
                         - $ref: '#/components/schemas/MarketplacePlugin'
                         - $ref: '#/components/schemas/GitMarketplacePlugin'
 
+  /publishing/packages:
+    get:
+      tags:
+        - publishing
+      summary: Get list of packages in the publishing queue
+      description: 'Required permission "get_publishing_queue"'
+      operationId: PublishingPackages
+      parameters:
+        - name: siteId
+          in: query
+          description: site ID
+          required: true
+          schema:
+            type: string
+        - name: environment
+          in: query
+          description: filter by publishing environment
+          required: false
+          schema:
+            type: string
+        - name: path
+          in: query
+          description: regular expression to filter by path
+          required: false
+          schema:
+            type: string
+        - name: state
+          in: query
+          description: filter by package state
+          required: false
+          schema:
+            type: string
+        - name: offset
+          in: query
+          description: Offset for pagination
+          required: false
+          schema:
+            type: integer
+            format: int64
+        - name: limit
+          in: query
+          description: Limit for pagination
+          required: false
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  packages:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/PublishingPackage'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /publishing/package_details:
+    get:
+      tags:
+        - publishing
+      summary: Get package details
+      description: 'Required permission "get_publishing_queue"'
```

Comment:
I'm okay with the same permission for both but want to make sure you meant it this way.

Reply:
In my mind, to read the publishing queue it is same permission whether it is list of packages or a single package.